lib/stat_tracker.rb

```diff
@@ -1,12 +1,17 @@
 require "csv"
 
 class StatTracker
+  attr_reader :games,
+              :teams,
+              :game_teams
 
-  def self.from_csv(locations_information)
-    @games = locations_information[:games]
-    @teams = locations_information[:teams]
-    @game_teams = locations_information[:game_teams]
-    self
+  def initialize(file_paths)
```

Comment:
We gave it attributes that lead to the file paths of the csv files for each of the game, teams and game_teams.

Reply:
Awesome!  Have some concerns about data validation, but I may be getting ahead of myself and AJ explained that Kathy, you share some of the same concerns.  So we'll cross that bridge when we get there.